.github/workflows/postprocessing.yml

```
@@ -0,0 +1,130 @@
+name: Post-processing
+
+
+# don't limit to main branch
+# consider maintenance branch
+on:
+  push:
+    branches:
+      - 'release-branch.**'
+      - main
```

```suggestion
  push:
    branches:
      - 'release-branch.**'
      - main
    paths-ignore:
      - '.github/**'
      - LICENSE
      - README.md
```

Do the same for the `verification.yml` workflow.

---

Done in [feat: add path ignore](https://github.com/goplus/llpkg/pull/14/commits/8fb5388e45b51cca101d28dcf62606194e7a7cab)